templates/named.conf.options.erb

```diff
@@ -1,6 +1,9 @@
 options {
-	directory "/var/cache/bind";
-
+<% if @osfamily == 'Debian' -%>
+  directory "/var/cache/bind";
+<% else -%>
+    directory "<%= @data_dir %>";
```

Comment:
Do we not have a data_dir on non-Debian oses? Can you make it not hard-coded in either case?